airflow/values.yaml

```diff
@@ -0,0 +1,57 @@
+# Default values for airflow-helm
+
+images: 
+  airflow: docker.io/eanylin/airflow-helm:1.8.1-1.6.2
+  pull_policy: "IfNotPresent"
+
+network:
+  flower:
+    port: 5555
+    node_port: 32081
+    enable_node_port: true
+  web:
+    port: 8080
+    node_port: 32080
+    enable_node_port: true
+
+replicas:
+  web: 1
+  worker: 1
+  flower: 1
+  scheduler: 1
+
+volumes:
+  path: /home/ubuntu/workbench/dags
+  
+scheduler_args: "5"
+
+resources:
+  enabled: false
+  worker:
```

Comment:
i see you've added this in the values.yaml but not in the manifests themselves -- go ahead and set a conditional in each of the deployment files